velox/exec/TableWriter.cpp

```diff
@@ -55,16 +53,16 @@ TableWriter::TableWriter(
   }
 
   mappedType_ = ROW(std::move(names), std::move(types));
-  const auto& config = driverCtx_->task->queryCtx()->config();
-  if (config.createEmptyFiles()) {
-    LOG(INFO) << "Enforce creating empty files\n";
-    createDataSink();
-  }
+  writeProtocol_ = connector::WriteProtocol::newWriteProtocol(commitStrategy);
+  createDataSink();
```

Comment:
Why is this change? Any particular reason we couldn't keep the existing logic?

Reply:
Changing it back